### ISSUE TITLE
Graphql variable causes runtime error

### ISSUE THREAD

**Issue by AbdelilahDerfoufi** [api-platform/api-platform#2332]
**API Platform version(s) affected**: 3.0.0

**Description**  

After updating php from 8.0 to 8.1 and symfony 5.4 to 6.1. to try to update apiplatform to version 3.0.0 I had issues such as a deleted api_platform.yaml from config/packages
and a runtime error

**How to reproduce**  

Update major versions of php and symfony to 8.1 and 6.1 respectively.

**Possible Solution**  

I commented the line 77 and it worked.
**Additional Context**  

![image](https://user-images.githubusercontent.com/49759161/204107371-11e38b21-41db-4226-8703-2c7c53272988.png)
![image](https://user-images.githubusercontent.com/49759161/204107410-6c4839fb-da31-4a74-9890-320dcc6793dd.png)

**Comment by AbdelilahDerfoufi** [comment]
> I' am also missing the api_platform.yaml config after updating from 6.0 to 6.1 and updating api-platform to version 3.0. Unfortunately the config is missing also after creating a new project with symfony 6.2 and api-platform 3.0. When adding it manually its settings dont seem to work.

Does the graphql causes your applicatiol to be down?

**Comment by ivan-hr** [comment]
Yes it did but I fixed it by editing the typo. Seel line 77 in my screenshot.
![image](https://user-images.githubusercontent.com/14945040/205652236-84cc1991-d575-4591-a9f1-3d9be53d7b2c.png)

**Comment by AbdelilahDerfoufi** [comment]
Then what is your error?

**Comment by ivan-hr** [comment]
I recoverd the missing api_platform.yaml but it gets ignored. None of the settings seems to work, I cant authenticate for example. Does this work for you?

**Comment by AbdelilahDerfoufi** [comment]
Can you check the directory?

**Comment by ivan-hr** [comment]
Its inside config/packages. You dont have problems with your projects api_platform.yaml? Recovered it and everything works fine?

**Comment by AbdelilahDerfoufi** [comment]
Yes, I created manually copied the original version and I put it in the new one. I updated my composer json as well. And update the reciepes also

**Comment by ivan-hr** [comment]
The config is in the right place... config/packages. But I just figured it out ... had to delete the cache folder! I started and stopped the server asuming that this would be enough. Thanks for input @AbdelilahDerfoufi!

**Comment by AbdelilahDerfoufi** [comment]
> The config is in the right place... config/packages. But I just figured it out ... had to delete the cache folder! I started and stopped the server asuming that this would be enough. Thanks for input @AbdelilahDerfoufi!

Oh yeah, cache always caused me problems in 2.7, I did delete the forlder and forgot all about it during the upgrade, sorry for not remembering.

**Comment by ivan-hr** [comment]
After you asked me if the directory was the correct one I remebered not deleting the cache folder. Glad you asked. Thanks again.

**Comment by AbdelilahDerfoufi** [comment]
No worries mate, glad I helped somehow.
